lib/hammer_cli_katello/content_view.rb

```diff
@@ -106,10 +106,12 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       failure_message _("Could not create the content view")
 
       option ["--composite"], :flag, _("Create a composite content view")
-
+      option ["--import-only"], :flag, _("Designate this Content View for "\
```

Comment:
Does this undo the server-side mechanism we use to conditionally show or hide the parameter?

Reply:
No, we do the same for composite as you see above. So trying to stay consistent with other boolean params